components/payments/internal/app/api/module.go

```diff
@@ -88,6 +98,18 @@ func handleServerError(w http.ResponseWriter, r *http.Request, err error) {
 	}
 }
 
+func handleNotFoundError(w http.ResponseWriter, r *http.Request, err error) {
```

Comment:
There is a common helper in the shared libs, maybe use it?

Reply:
Done !